lib/bottom_picker.dart

```diff
@@ -242,6 +242,60 @@ class BottomPicker extends StatefulWidget {
     onChange = null;
     onSubmit = null;
     displaySubmitButton = true;
+    use24hFormat = true;
+    assert(onRangeDateSubmitPressed != null);
+    assertInitialValues();
+    if (minSecondDate != null && initialSecondDate != null) {
+      assert(initialSecondDate!.isAfter(minSecondDate!));
+    }
+    if (minFirstDate != null && initialFirstDate != null) {
+      assert(initialFirstDate!.isAfter(minFirstDate!));
+    }
+  }
+
+  BottomPicker.rangeTime({
+    Key? key,
+    required this.pickerTitle,
+    this.pickerDescription,
+    required this.onRangeDateSubmitPressed,
+    this.use24hFormat = true,
+    this.titlePadding = const EdgeInsets.all(0),
+    this.titleAlignment,
+    this.dismissable = false,
+    this.onClose,
+    this.bottomPickerTheme = BottomPickerTheme.blue,
+    this.gradientColors,
+    this.buttonPadding,
+    this.buttonWidth,
+    this.buttonSingleColor,
+    this.backgroundColor = Colors.white,
+    this.pickerTextStyle = const TextStyle(
+      fontSize: 14,
+      color: Colors.black,
+    ),
+    this.displayCloseIcon = true,
+    this.closeIconColor = Colors.black,
+    this.closeIconSize = 20,
+    this.layoutOrientation = TextDirection.ltr,
+    this.buttonAlignment = MainAxisAlignment.center,
+    this.height,
+    this.initialSecondDate,
+    this.initialFirstDate,
+    this.minFirstDate,
+    this.minSecondDate,
+    this.maxFirstDate,
+    this.maxSecondDate,
```

Comment:
for all "Date" attributes either replace them with Time attributes for more coherent usage and naming or simply remove them and we can add them in later version
```
this.initialSecondDate,
this.initialFirstDate,
this.minFirstDate,
this.minSecondDate,
this.maxFirstDate,
this.maxSecondDate,
```

Reply:
I changed all Date to TIme